docs/tutorials/content.md

```diff
@@ -0,0 +1,61 @@
+## Itowns' basics
+
+### 1 - Introduction
+
+### 2 - The `View` : a support for data visualization
+
+### 3 - Data visualization process
+
+#### A) Fetching data
+
+#### B) Parsing data
+
+#### C) Converting data
+
+### 5 - Vector data processing
```

Comment:
merge in one part

Fetch -> Parse -> Convert

Add example to create a simple Parser.